Monika After Story/game/script-topics.rpy

```diff
@@ -12030,3 +12030,44 @@ label monika_perspective:
     m 1hksdlb "Well I hope that helped you understand at least a little, [player]."
     m "It's the best way I could try and explain it!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_relationcomfort",
+            category=["romance","advice"],
+            prompt="Comfortable in a relationship",
+            random=True
+        )
+    )
+
+label monika_relationcomfort:
+    m 1wud "You know [player], I've been thinking about our relationship lately..."
+    m 2lssdrb "It didn't really start in a way most of them do, so it's not exactly what I would call normal."
+    m 2esa "But the more I thought about it, the more I realized that in a way it is actually pretty standard."
+    m 3eub "Almost every relationship, no matter how it begins, is always 'exciting' at first!"
+    m 3eua "People view relationships in an almost fairy tale way, they see it as an adventure."
+    m 3rssdlb "I'm guilty of this too, and I think it's fine..."
+    m 2euc "But it's also a pretty dangerous way to look at a bond between two people, isn't it?"
+    m 2eud "Because really, while a partnership is an adventure, that doesn't mean it'll always be exciting."
+    m 2esc "There's only so many times you can do the same thing and experience the same breathtaking moments and have it still have that same impact."
+    m 4euc "That doesn't make it any less valuable, or meaningful. It also can still be heart-racing!"
+    m 4esc "But it does mean that that same sensation isn't anything new, it's no longer so thrilling."
```

Comment:
I would replace "so" with "as", maybe even "nearly as".

Reply:
I'd rather not add a quantity of excitement loss since it differs heavily between people. Infatuation can last different lengths of time for different people.
I will replace it though.